rpc/handlers.go

```diff
@@ -221,18 +221,20 @@ func (h *Handler) TransactionByHash(hash *felt.Felt) (*Transaction, *jsonrpc.Err
 	return adaptTransaction(txn), nil
 }
 
-// BlockTransactionCount https://github.com/starkware-libs/starknet-specs/blob/a789ccc3432c57777beceaa53a34a7ae2f25fda0/api/starknet_api_openrpc.json#L373
-func (h *Handler) BlockTransactionCount(id *BlockId) (uint64, *jsonrpc.Error) {
-	header, err := h.blockHeaderById(id)
+// BlockTransactionCount
+//
+// https://github.com/starkware-libs/starknet-specs/blob/a789ccc3432c57777beceaa53a34a7ae2f25fda0/api/starknet_api_openrpc.json#L373
```

Comment:
I understand that the function names are descriptive. But can we follow this convention for the functions here?
```suggestion
// BlockTransactionCount returns the number of transactions in a block
// identified by the given BlockID.
//
// It follows the specification found here:
// https://github.com/starkware-libs/starknet-specs/blob/a789ccc3432c57777beceaa53a34a7ae2f25fda0/api/starknet_api_openrpc.json#L373
```

Reply:
Or maybe in another PR since this is just for fixing linting errors.

Reply:
In another pr would be best. I don't want to rebase 17 commits